src/ape/plugins/_utils.py

```diff
@@ -361,7 +361,7 @@ def _prepare_install(
             logger.warning(f"Plugin '{self.name}' is not an trusted plugin.")
 
         result_handler = ModifyPluginResultHandler(self)
-        pip_arguments = [sys.executable, "-m", "pip", "install"]
+        pip_arguments = PIP_COMMAND + ["install"]
```

Comment:
style nit

```suggestion
        pip_arguments = [*PIP_COMMAND, "install"]
```

Reply:
applied same style in 2 other places in bd28b327ef62e28a40a526ae4a31240c30c1ddc4